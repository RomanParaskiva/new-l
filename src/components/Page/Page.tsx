import React from "react";

import { Header } from "../Header/Header";
import { Field } from "../Field/Field";
import { Sidebar } from "../Sidebar/Sidebar";
// import { LoginForm } from "../LoginForm/LoginForm";
import "./page.css";

export const Page: React.FC = () => {
  return (
    <div className="h-screen">
      <Header />

      <section className="h-full flex gap-5">
        <Sidebar />
        <Field />
        {/* <LoginForm /> */}
      </section>
    </div>
  );
};
