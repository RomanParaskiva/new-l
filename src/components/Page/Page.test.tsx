import { render } from "@testing-library/react";

import { Page } from "./Page";

describe("test Page component", () => {
  test("renders Page component", () => {
    const { container } = render(<Page />);

    expect(container).toMatchSnapshot();
  });

});
