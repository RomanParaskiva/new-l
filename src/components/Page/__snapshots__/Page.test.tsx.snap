// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Page component renders Page component 1`] = `
<div>
  <div
    class="h-screen"
  >
    <header>
      <div
        class="wrapper"
      >
        <div>
          <h1>
            Game of Life
          </h1>
        </div>
      </div>
    </header>
    <section
      class="h-full flex gap-5"
    >
      <div>
        <div
          class="mx-auto flex flex-col w-[300px] gap-3 bg-slate-100 rounded-2xl p-3"
        >
          <fieldset
            class="flex flex-col gap-2"
          >
            <button
              class="p-2 w-full text-sm rounded-xl uppercase outline-none text-gray-600 hover:scale-105 font-semibold bg-green-300"
            >
              Старт
            </button>
            <button
              class="p-2 w-full text-sm rounded-xl uppercase outline-none text-gray-600 hover:scale-105 font-semibold bg-blue-300"
            >
              Сброс
            </button>
          </fieldset>
          <fieldset
            class="flex flex-col gap-2"
          >
            <legend>
              Скорость
            </legend>
            <button
              class="p-2 w-full text-sm rounded-xl uppercase outline-none text-gray-600 hover:scale-105 font-semibold bg-blue-300"
            >
              1
            </button>
            <button
              class="p-2 w-full text-sm rounded-xl uppercase outline-none text-gray-600 hover:scale-105 font-semibold bg-blue-300"
            >
              1.5
            </button>
            <button
              class="p-2 w-full text-sm rounded-xl uppercase outline-none text-gray-600 hover:scale-105 font-semibold bg-blue-300"
            >
              2
            </button>
          </fieldset>
          <fieldset
            class="flex flex-col gap-2"
          >
            <legend>
              Размеры
            </legend>
            <button
              class="p-2 w-full text-sm rounded-xl uppercase outline-none text-gray-600 hover:scale-105 font-semibold bg-purple-300 disabled:bg-gray-400 disabled:text-gray-600"
            >
              50/50
            </button>
            <button
              class="p-2 w-full text-sm rounded-xl uppercase outline-none text-gray-600 hover:scale-105 font-semibold bg-purple-300 disabled:bg-gray-400 disabled:text-gray-600"
            >
              30/30
            </button>
            <button
              class="p-2 w-full text-sm rounded-xl uppercase outline-none text-gray-600 hover:scale-105 font-semibold bg-purple-300 disabled:bg-gray-400 disabled:text-gray-600"
            >
              100/100
            </button>
          </fieldset>
          <fieldset>
            <legend>
              Процент заполнености
            </legend>
            <div
              class="relative pt-1"
            >
              <label
                class="form-label"
                data-testid="rangeLabel"
                for="customRange1"
              >
                0 %
              </label>
              <input
                class="form-range appearance-none w-full h-3 rounded-lg p-0 bg-blue-100 focus:outline-none focus:ring-0 focus:shadow-none"
                data-testid="rangeInput"
                id="customRange1"
                max="80"
                min="0"
                type="range"
                value="0"
              />
            </div>
          </fieldset>
        </div>
      </div>
      <div
        class="sc-bczRLJ lijUsN"
      />
    </section>
  </div>
</div>
`;
